post_installation_configuration/aws-compute-edge-tasks.adoc

```diff
@@ -0,0 +1,75 @@
+:_content-type: ASSEMBLY
+[id="aws-compute-edge-tasks"]
+= AWS Local Zone tasks
+include::_attributes/common-attributes.adoc[]
+:context: aws-compute-edge-tasks
+
+toc::[]
+
+After installing {product-title} on Amazon Web Services (AWS),
+you can further configure AWS Local Zones and an edge compute
+pool, so that you can expand and customize your cluster to meet your
+requirements.
+
+// Day-2 Section: Extend existing clusters to use AWS Local Zones
+include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features]
+in the AWS documentation for more information about AWS Local Zones and the supported instances types and services.
+
+////
+// About the Edge Compute Pool - Need to ask Marco
+include::modules/machines-edge-machine-pool.adoc[leveloffset=+2]
+////
+
+[id="extend_existing_to_local_zones_day2_mtu"]
+=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+
+You might need to change the maximum transmission unit (MTU) value for the cluster
+network so that your cluster infrastructure can support Local Zone subnets.
+
+// MTU
+include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
```

Comment:
How about I add this module as either an additional resource link or as a prerequisite in nw-cluster-mtu-change.adoc ?